tokio/src/sync/mpsc/bounded.rs

```diff
@@ -849,10 +861,68 @@ impl<T> Sender<T> {
     /// }
     /// ```
     pub async fn reserve(&self) -> Result<Permit<'_, T>, SendError<()>> {
-        self.reserve_inner().await?;
+        self.reserve_inner(1).await?;
         Ok(Permit { chan: &self.chan })
     }
 
+    /// Waits for channel capacity. Once capacity to send `n` message is
```

Comment:
Please make sure we use the plural form in the docs adapted from `reserve` where applicable.
```suggestion
    /// Waits for channel capacity. Once capacity to send `n` messages is
```